components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/RequestObjectValidatorUtil.java

```diff
@@ -245,4 +289,22 @@ public static boolean isSignatureVerified(SignedJWT signedJWT, Certificate x509C
             return false;
         }
     }
+
+    public static JWSVerifier getVerifier(PublicKey publicKey) {
+
+        return new RSASSAVerifier((RSAPublicKey) publicKey);
+
```

Comment:
remove empty line

Reply:
Fixed in f409e32a5e8cad00a5dfeaaac027011720f2c534